Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.py

```diff
@@ -0,0 +1,1791 @@
+        'inProgress': event.get('InProgress'),
+        'url': event.get('Url'),
+        'input': event.get('Input'),
+        'sourceEmailAddress': event.get('Source_Email_Address'),
+        'agentless': event.get('Agentless'),
+        'analysis': event.get('Analysis'),
+        'command': event.get('Command'),
+        'description': extract_event_description(event),
+        'md5': event.get('MD5') if event.get('MD5') else event.get('Md5'),
+        'sha256': event.get('SHA256') if event.get('SHA256') else event.get('Sha256'),
+        'sha1': event.get('SHA1') if event.get('SHA1') else event.get('Sha1'),
+    }
+
+    mitigation = event.get('Mitigation_Details') if event.get('Mitigation_Details') else event.get(
+        'Mitigation')
+    if mitigation:
+        incident_data['mitigationDetails'] = mitigation
+
+    attack_payload = event.get('Attack_Payload')
+    if attack_payload and attack_payload.startswith('http'):
+        incident_data['url'] = attack_payload
+
+    elif attack_payload:
+        incident_data['attackType'] = attack_payload
+
+    incident = {
+        'name': extract_event_name(event, module_name),
+        'occurred': event_timestamp,
+        'severity': convert_to_xsoar_severity(event.get('Risk')),
+        'rawJSON': json.dumps(remove_empty_elements(incident_data)),
+    }
+
+    if not event_timestamp:
+        event_timestamp = event.get('Timestamp') if event.get('Timestamp') \
+            else event.get('Attack_Timestamp')
+    if validate_timestamp(event_timestamp):
+        occurred = datetime.strptime(event_timestamp, CY_GENERAL_DATE_FORMAT).strftime(DATE_FORMAT)
+        incident['occurred'] = occurred
+
+    return incident
+
+
+def convert_to_xsoar_severity(severity: Optional[Any]) -> int:
+    """Maps Cymulate severity to Cortex XSOAR severity.
+
+    Args:
+        severity(str): severity as returned from Cymulate API.
+
+    Returns:
+        int: Cortex XSOAR Severity (1 to 4)
+    """
+    if severity:
+        return {
+            'low': IncidentSeverity.LOW,
+            'medium': IncidentSeverity.MEDIUM,
+            'high': IncidentSeverity.HIGH,
+            'critical': IncidentSeverity.CRITICAL
+        }.get(severity.lower(), 0)
+    return 0
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication. Returning 'ok' indicates that the integration
+    works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client (Client): Cymulate client.
+    """
+    test_message = 'ok'
+    try:
+        response = client.validate()
+        if not response.ok:
+            raise DemistoException('Authorization Error: make sure API Key is correctly set.')
+
+    except DemistoException as err:
+        if 'Unauthorized' in str(err):
+            test_message = f'Authorization Error: make sure API Key is correctly set.\n\n{err}'
+        else:
+            raise err
```

Comment:
raising the exception object again will reset the traceback to this location and mask the actual error location.
using an empty raise statement inside an except scope will raise the original exception and keep the traceback as expected.
```suggestion
            raise
```